iteration/iteration_test.go

```diff
@@ -44,17 +44,27 @@ func (s *TestIterationRepository) TestCreateIteration() {
 			EndAt:   &end,
 		}
 		// when
-		repo.Create(context.Background(), &i)
+		err := repo.Create(context.Background(), &i)
 		// then
-		if i.ID == uuid.Nil {
-			t.Errorf("Iteration was not created, ID nil")
-		}
-		if i.CreatedAt.After(time.Now()) {
-			t.Errorf("Iteration was not created, CreatedAt after Now()?")
-		}
+		require.NoError(t, err)
+		require.NotEqual(t, uuid.Nil, i.ID, "iteration not created, ID is nil")
+		// require.True(t, i.CreatedAt.After(time.Now()), "iteration was not created, CreatedAt after Now()?")
```

Comment:
Should we add this require back?

Reply:
Good question. Will try.

Reply:
Fixed in 3b94546897c3c3fae6cec46a6d0a078300e31f11